SDJSBridge/JavaScript API/Platform API/Navigation/SDJSNavigationScript.m

```diff
@@ -105,7 +105,11 @@ - (void)pushStateWithOptions:(NSDictionary *)options {
 
 - (void)replaceStateWithOptions:(NSDictionary *)options {
     @strongify(self.webViewController, strongWebViewController);
-    strongWebViewController.title = options[kSDJSNavigationScriptTitleKey];
+    
+    NSString *title = options[kSDJSNavigationScriptTitleKey];
+    if (title.length) {
+        strongWebViewController.title = options[kSDJSNavigationScriptTitleKey];
```

Comment:
I would just reuse the value in the `title` pointer instead of calling objectForKey: again.

```
strongWebViewController.title = title;
```